src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -1,7 +1,25 @@
 <script>/*<![CDATA[*/
+    {# Caution: the contents of this file are processed by Twig before loading
+                them as JavaScript source code. Always use '/*' comments instead
+                of '//' comments to avoid impossible-to-debug side-effects #}
+
     Sfjs = (function() {
         "use strict";
 
+        var classListIsSupported = 'classList' in document.createElement('_');
```

Comment:
creating a div would be better than creating a _ element IMO (which could do weird things)